.editorconfig

```
@@ -1,6 +1,6 @@
 root = true
 
-[*.ts,*.js]
+[*{.ts,*.js}]
```

Should this be the below?
```suggestion
[*{.ts,.js}]
```

---

Probably the below would be better?
```suggestion
[*.{ts,js}]
```